modules/hub/pages/relay-mode.adoc

```diff
@@ -0,0 +1,54 @@
+[[manual-mode-example]]
+= Manual mode
+
+
+In this example, relay mode is used, this is a convenient method where it is assumed that each server has an admin user with the same credentials as one of Hub's user but user still needs to call `attachToServers` method.
```

Comment:
I would reword it as something like:

The `Relay mode` is a convenient mode where the same credentials used to login into the Hub API will be used to login into the server's APIs as well the user wants to work with. It is assumed that the credentials are valid in each server and correspond to an admin user. After logging in into the Hub API the user still needs to call `attachToServers` method.